src/hooks/useNotificationTypes.tsx

```diff
@@ -413,6 +415,90 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.MarketWindDown,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+
+      const marketWindDownProposalExpirationDate = '2024-06-11T16:53:00';
+      const marketWindDownDate = marketWindDownProposalExpirationDate;
+      const marketWindDownExpirationDate = '2024-07-11T16:53:00'; // 30 days after wind down
+      const currentDate = new Date();
+
+      const outputDate = <$Output type={OutputType.DateTime} value={marketWindDownDate} />;
+
+      const firstMarket = 'FET-USD';
+      const secondMarket = 'AGIX-USD';
+
+      useEffect(() => {
+        if (currentDate <= new Date(marketWindDownProposalExpirationDate)) {
+          trigger(MarketWindDownNotificationIds.MarketWindDownProposalFetAgix, {
+            title: stringGetter({
+              key: 'NOTIFICATIONS.TWO_MARKET_WIND_DOWN_PROPOSAL.TITLE',
+              params: {
+                MARKET_1: firstMarket,
+                MARKET_2: secondMarket,
+              },
+            }),
+            body: stringGetter({
+              key: 'NOTIFICATIONS.TWO_MARKET_WIND_DOWN_PROPOSAL.BODY',
+              params: {
+                MARKET_1: firstMarket,
+                MARKET_2: secondMarket,
+                DATE: outputDate,
+                HERE_LINK: (
+                  <$Link href="https://www.mintscan.io/dydx/proposals/61">
```

Comment:
should these go into links config?

Reply:
sooo mobile won't have these notifications cc @prashanDYDX so I think it's ok to leave them here? (esp since they're time bound notifications, but lmk if I'm wrong!)